pandas/io/excel.py

```diff
@@ -168,7 +168,20 @@ def get_writer(engine_name):
         raise ValueError("No Excel writer '%s'" % engine_name)
 
 
-@Appender(_read_excel_doc)
+def split_doc(doc_string, word_wrap=80):
```

Comment:
no this is adding code which is the same as wrap
otherwise it needs to be in a common location